core/src/main/java/org/akaza/openclinica/bean/extract/odm/MetaDataReportBean.java

```diff
@@ -138,7 +138,7 @@ private String handleLoadCastor(RulesPostImportContainer rpic) {
             String newResult = result.replace("<?xml version=\"1.0\" encoding=\"UTF-8\"?>", "");
             // The result of this is being double escaped. Somewhere the Castor library is escaping the XML specific
             // characters when they have already been escaped.
-            newResult = StringEscapeUtils.unescapeXml(newResult);
```

Comment:
did you try removing this line completely? From the comment, it sounds like the result was being double escaped already.